packager/CMakeLists.txt

```diff
@@ -86,6 +86,13 @@ else()
   add_library(libpackager STATIC ${libpackager_sources})
 endif()
 
+set(LIBPACKAGER_HEADER_FILES
+  file/file.h
+  macros.h
+  packager.h
```

Comment:
Yeah, this is definitely not a complete set of headers.  You can just look at the list of `#include`s in `packager.h`.

I'll create a test for this.